README.md

```diff
@@ -1,16 +1,16 @@
 # CodeIgniter 4 Development
 
-[![Build Status](https://travis-ci.org/codeigniter4/CodeIgniter4.svg?branch=develop)](https://travis-ci.orgcodeigniter4/CodeIgniter4)
-[![Coverage Status](https://coveralls.io/repos/github/codeigniter4/CodeIgniter4/badge.svg?branch=develop)](https://coveralls.io/githubcodeigniter4/CodeIgniter4?branch=develop)
+[![Build Status](https://travis-ci.org/bcit-ci/CodeIgniter4.svg?branch=develop)](https://travis-ci.org/bcit-ci/CodeIgniter4)
```

Comment:
the link was missing `/` ( "travis-ci.orgcodeigniter4" ), I can change to "travis-ci.org/CodeIgniter4" if the url is the correct one. But previously, it will definatelly got unknown page in browser ( unable to connect  ), cc @jim-parry

Reply:
That is on purpose. I haven't updated travis-ci.org yet ... trying to do this in a reasonable sequence :-/
I need to update packagist.org too!
Step 1: the doc changes, 2 - transfer the repo, 3 - update travis & packagist, 4 - admin build testing.
Thank you for being diligent, though!